alpha-blog/app/views/articles/_article.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <% obj.each do |article| %>
```

Comment:
What is `obj` ?
Naming is super important as it tells us (the readers) what it represents